spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -414,6 +416,93 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
     });
   });
 
+  describe('choosing and unchoosing', () => {
+    beforeEach(() => {
+      omniboxController.ngModel = ['one', 'two'];
+    });
+
+    it('should set the ngModel to the choice when multiple is off', () => {
+      omniboxController.choose('three');
+
+      expect(omniboxController.ngModel).toEqual('three');
+    });
+
+    it('should push the choice to an array when multiple is on', () => {
+      omniboxController.multiple = true;
```

Comment:
@wei Ok, so what's more palatable to you? Dropping the `Array.isArray` checks when trying to push and instead just checking multiple, and if they didn't make it an array when they set it to multiple to just let it error out? Or to try to force the ngModel to an array if we find multiple to be true but it's not currently an array?